src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { TabsInterface } from '../../Types/TabsInterface';
+
+type Props = {
+  tabs: TabsInterface[];
+  onTabSelected: TabsInterface;
```

Comment:
```suggestion
  selectedTab: TabsInterface;
```
`onTabSelected` looks like it should take a callback